cheatsheet.md

```diff
@@ -200,6 +198,9 @@ chai.eval(R"_(
   )_");
 ```
 
+## Adding STL math
+ChaiScript itself does not provide a link to the math functions defined in `<cmath>`. You can either add them yourself, or use the [https://github.com/ChaiScript/ChaiScript_Extras](ChaiScript_Extras) helper library. (Which also provides some additional string functions.)
```

Comment:
Rather than just referencing STL math here, we could make it a generic `## Extras` docs near the bottom of the docs.

Reply:
Okay. I thought it would match with the STL types here. Do you want it changed?

Plus, how do I actually do that? I never had parts of my pull request updated before 😆 Do I  just submit another patch?

Reply:
Just commit in the same branch and the PR will be updated :wink: